container/root/etc/nginx/nginx.conf

```diff
@@ -16,12 +16,66 @@ pid /tmp/.nginx/nginx.pid;
 
 error_log /dev/stdout warn;
 
+# Number of file descriptors used for nginx
+worker_rlimit_nofile 40000;
```

Comment:
in theory, yes. but I can't think of much of a reason to modify it at runtime. instead, a downstream client would simply `sed` it out